mlrun/execution.py

```diff
@@ -855,22 +862,27 @@ def commit(self, message: str = "", completed=True):
         if self._parent:
             self._parent.update_child_iterations()
             self._parent._last_update = now_date()
-            self._parent._update_db(commit=True, message=message)
```

Comment:
not a fan of the one liner above `completed = completed and self._state == "running"` please change it make it more explainable